src/pages/Components/FileUploader/specs.md

```diff
@@ -7,9 +7,39 @@ order: 3
 
 => anatomy image
 
-### define values
+## Properties
 
-- tokens tables / properties
-- schematics (with sizes margins etc)
-- margins, units, styles etc
-- same for responsive behaviors
\ No newline at end of file
+### Layout / size
+
+ Height       | Paddings     | Font-size             | radius           |
+ ------------ | ------------ | --------------------- | ---------------- |
+ 48px (3mu)   | 32px (2mu)   | `size.font.05` (16px) | `radius.m` (4px) |
+
+
+### Icon / size
+
+| Width and height                   |
+| ---------------------------------- |
+| 24px (1.5mu)                       |
+
+
+### Color theme
+
+File uploader states colors according to the theme of the `bordered` buttons :
+
+| State              | Property         | Token                                       |
+| ------------------ | ---------------- | ------------------------------------------- |
+| **Default**        | background-color | `color.button.bordered.background`          |
+| **Default**        | color            | `color.button.bordered.font`                |
+| **Default**        | border-color     | `color.button.bordered.border`              |
+| **Hover**          | background-color | `color.button.bordered.hover.background`    |
+| **Hover**          | color            | `color.button.bordered.hover.font`          |
+| **Focus**          | border-color     | `color.button.bordered.focus.border`        |
+| **Active**         | background-color | `color.button.bordered.active.background`   |
+| **Disabled**       | background-color | `color.button.bordered.disabled.background` |
+| **Disabled**       | color            | `color.button.bordered.disabled.font`       |
+
+Except for the colors of icons :
+
+| **Default**        | color            | `color.fileuploader.default`                |
```

Comment:
It miss a head for the table
```html
| xxx                      | xxx                    | xxx                                                          |
| ------------------ | ---------------- | ------------------------------------------- |
```

![image](https://user-images.githubusercontent.com/43854639/63838804-59bafe00-c97e-11e9-80f1-5a680e226499.png)